.github/workflows/test-macos.yml

```
@@ -81,5 +81,10 @@ jobs:
           echo "::endgroup::"
       - name: Build
         run: make build-ci -j$(getconf _NPROCESSORS_ONLN) V=1 CONFIG_FLAGS="--error-on-warn"
+      - name: Free Space After Build
+        run: |
+          echo "::group::Free space after build"
+          df -h
+          echo "::endgroup::"
```

Since we only want to check the remaining space perhaps could be just one line.
```suggestion
        run: df -h
```